cli/pcluster/config/param_types.py

```diff
@@ -675,6 +683,59 @@ def to_file(self, config_parser):
         pass
 
 
+class DisableHyperThreading(Param):
+    """
+    Class to manage the disable_hyperthreading configuration parameter.
+
+    We need this class in order to convert the boolean disable_hyperthreading = [true/false] into Cores.
+    """
+
+    def from_cfn_params(self, cfn_params):
+        """Initialize param value by parsing the right CFN input."""
+        cfn_converter = self.definition.get("cfn_param_mapping", None)
+        if cfn_converter and cfn_params:
+            cores = int(get_cfn_param(cfn_params, cfn_converter).split(",")[0])
```

Comment:
You need a test to verify the value returned from `get_cfn_param` because if the param is not present or NONE (e.g. an old cluster without the CFN parameter) we can have an `IndexError`.